lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -103,6 +138,18 @@
     });
   };
 
+  // Makes a reasonable determination that
+  // the data passed to the function is formatted
+  // JSON and returns `true`, if so. False, if not
+
+  function isJSON (json, callback) {
+    if (!json) {
+      return callback(new Error('You forgot to pass JSON data as the first argument.'));
+    }
+    var cases = [ true, false, null, '"', '-', '[', '{' ];
```

Comment:
You're looking for whether `json` is a string that begins with one of these strings - `'true'`, not `true`, for instance.